nncf/quantization/algorithms/hyperparameter_tuner/param_grid.py

```diff
@@ -70,5 +74,38 @@ def get_quantization_param_grid() -> Dict[str, Any]:
             for min_v, max_v in itertools.product(min_param_values, max_param_values)
         ],
     }
-
     return param_grid
+
+
+def _get_smooth_quant_param_grid() -> ParamGrid:
+    return {"advanced_parameters:smooth_quant_alpha": [0.2, 0.5, 0.9, 0.95]}
```

Comment:
Is it possible to set first parameter 0.15 and make more "symmetric" grid: [0.15, 0.25, 0.5, 0.75, 0.95] ?

Reply:
Sure! I used these values only for testing purposes. I updated them.